project-infrastructure/project-ci-documentation.md

```diff
@@ -78,23 +78,42 @@ Representative: Timmie Smith
 
 Support contact for CI:
 
+| Maintainers               |
+| ------------------------- |
+| Dan Hoeflinger @danhoeflinger |
+| Dmitriy Sobolev @dmitriy-sobolev |
+| Timmie Smith @timmiesmith |
+
 *Existing public CI*
```

Comment:
I think it would be nice to mention how to run this CI, e.g. that it is automatically triggered in oneDPL repository when a PR is opened and a new commit is made.

I am curious how it should be used/triggered throughout forks, though.

Reply:
This is too much detail for what is being requested at this time.

Reply:
Done.